joomla/joomla-cms
# [4.0] [Backend] In Banners: Clients details page Contact column is di…

…splaying blank value

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/31355  // added by JM
 
### Summary of Changes


### Testing Instructions
1. log in to the backend
2. Goto Components-> Banners -> Clients
3. Click the New button and fill the form
4. While filling the form add tabs/spaces in "Contact Name" which is the required field.
5. Click on Save and close button
6. You can see the contact column for our added client is blank.

### Actual result BEFORE applying this Pull Request


### Expected result AFTER applying this Pull Request



### Documentation Changes Required
